README.md

```diff
@@ -3,26 +3,34 @@ The Azure App Services extension for VS Code lets you quickly browse, create, ma
 
 ## Features
 
-* Browse sites across all of your Azure subscriptions
-* Browse to the Azure Portal for advanced tasks, such as scaling
-* Create new web apps/deployment slots (Linux with Node.js only)
-* Deploy to your web apps/deployment slots
+  * Browse sites across all of your Azure subscriptions
```

Comment:
Don't add these spaces. It goes from this:
![screen shot 2017-12-15 at 12 03 36 pm](https://user-images.githubusercontent.com/11282622/34058492-15d93b20-e190-11e7-8321-fc1305a7b59f.png)

To this:
![screen shot 2017-12-15 at 12 03 31 pm](https://user-images.githubusercontent.com/11282622/34058502-1abff7d2-e190-11e7-8c9a-f3821b515903.png)

(Too much whitespace)

Reply:
Hmm, I'm not sure that fixed it..